### PR TITLE
TG-1700: Add a new designed icon for apps/projects

### PR DESCRIPTION
**Pull request by jehadmelad**
TG-1700

### CODE REVIEW

File: src/app/modals/launchpad/launch-pad.view.ts [hunk 1]
```diff
@@ -409,7 +432,13 @@ export class RunningAppView implements VirtualDOM {
             class: `w-100 fv-text-primary d-flex align-items-center justify-content-between position-relative yw-minimized-app`,
             children: [
                 child$(this.executable.appMetadata$, (d) => ({
-                    class: 'p-2',
+                    class: 'p-1',
+                    style: {
+                        height: '40px',
+                        width: '40px',
+                        // borderRadius: '5px',
```
Comment (youwol-guillaume) [inline src/app/modals/launchpad/launch-pad.view.ts:439]: Can we remove those commented lines of code?

File: src/app/top-banner/running-app-banner.view.ts [hunk 2]
```diff
@@ -89,12 +89,20 @@ class RunningAppTitleView implements VirtualDOM {
...
+                        return {
+                            style: {
+                                height: '25px',
+                                width: '25px',
+                                borderRadius: '5px',
+                            },
+                            children: [appInfo.graphics.appIcon],
+                        }
+                    }),
                     // { class: 'mx-1' },
```
Comment (youwol-guillaume) [inline src/app/top-banner/running-app-banner.view.ts:102]: to remove?

**Review by youwol-guillaume — DISMISSED** [review]

File: src/app/top-banner/running-app-banner.view.ts (hunk 2 above)
Comment (jehadmelad) [reply src/app/top-banner/running-app-banner.view.ts:102]: ok, it will be done.

**Review by youwol-guillaume — APPROVED** [review]

**Comment by jehadmelad** [comment]
TG-1700 #closed

**Pull request merged**